repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -287,59 +383,73 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
     }
 
+    updatePath match {
+      case Success(newLink) => newLink
+      case Failure(e: PathRewriteException) =>
+        failureBehavior match {
+          case CONTINUE => link
+          case REMOVE => DROP_CODE
+          case FAIL => throw e
+        }
+      case Failure(e) => throw e
+    }
+  }
+
   override def configurationUpdated(uriStripperConfig: UriStripperConfig): Unit = {
     val saxonTransformFactory = {
       val f = TransformerFactory.newInstance("net.sf.saxon.TransformerFactoryImpl", this.getClass.getClassLoader)
       val cast = f.asInstanceOf[TransformerFactoryImpl]
       cast.getConfiguration.getDynamicLoader.setClassLoader(this.getClass.getClassLoader)
       // Recover silently from recoverable errors. These may occur depending on XPath passed in.
-      f.setAttribute("http://saxon.sf.net/feature/recoveryPolicyName","recoverSilently")
+      f.setAttribute("http://saxon.sf.net/feature/recoveryPolicyName", "recoverSilently")
       f
     }
 
     val setupTemplate = saxonTransformFactory.newTemplates(new StreamSource(getClass.getResource("/META-INF/schema/xslt/transformer.xsl").toString))
 
-    def setupTransformer(xmlElement: HttpMessage.Xml): Templates = {
+    def setupTransformer(xmlElement: HttpMessage.Xml, isRequest: Boolean): Templates = {
       val setupTransformer = setupTemplate.newTransformer
       setupTransformer.setParameter("xpath", xmlElement.getXpath.getValue)
       setupTransformer.setParameter("namespaces", new StreamSource(
         <namespaces xmlns="http://www.rackspace.com/repose/params">
-          {
-          for (namespace <- xmlElement.getNamespace) yield
-              <ns prefix={namespace.getName} uri={namespace.getUrl}/>
-          }
+          {for (namespace <- xmlElement.getNamespace) yield
+            <ns prefix={namespace.getName} uri={namespace.getUrl}/>}
         </namespaces>
       ))
-      setupTransformer.setParameter("failOnMiss", (xmlElement.getXpath.getLinkMismatchAction == FAIL))
+      setupTransformer.setParameter("failOnMiss", xmlElement.getXpath.getLinkMismatchAction == FAIL)
       setupTransformer.asInstanceOf[Controller].addLogErrorListener
       val updateXPathXSLTDomResult = new DOMResult()
-      setupTransformer.transform (new StreamSource(<ignore-input />), updateXPathXSLTDomResult)
+      setupTransformer.transform(new StreamSource(<ignore-input/>), updateXPathXSLTDomResult)
 
-      val config = saxonTransformFactory.asInstanceOf[TransformerFactoryImpl].getConfiguration()
-      config.registerExtensionFunction(new UrlPathTransformExtension("process-url","rax", "http://docs.rackspace.com/api",
-        xsltFunction(Option(xmlElement.getXpath.getTokenIndex).map(_.intValue()), xmlElement.getXpath.getLinkMismatchAction)))
+      val config = saxonTransformFactory.asInstanceOf[TransformerFactoryImpl].getConfiguration
+      config.registerExtensionFunction(new UrlPathTransformExtension("process-url", "rax", "http://docs.rackspace.com/api",
```

Comment:
Some of these were caught by the static analysis tools and some were updates on a full format check.